BraveWallet/Settings/CustomNetworkListView.swift

```diff
@@ -86,6 +88,12 @@ struct CustomNetworkListView: View {
         }
       }
       .listRowBackground(Color(.secondaryBraveGroupedBackground))
+      
+      Section {
+        Toggle(Strings.Wallet.showTestNetworksTitle, isOn: $showTestNetworks.value)
+          .foregroundColor(Color(.braveLabel))
+          .toggleStyle(SwitchToggleStyle(tint: Color(.braveBlurpleTint)))
+      }
     }
```

Comment:
since now we have two sections, can we check if `customNetworks.isEmpty` first to create the first section? currently the first section always there makes the test network toggle section has some extra padding on top.
` let customNetworks = networkStore.ethereumChains.filter({ $0.isCustom })
      if !customNetworks.isEmpty {
      // custom network row
}
Section {
     // test networks toggle
}
`

Reply:
Nice thanks, fixed here https://github.com/brave/brave-ios/pull/5656/commits/4d9c520f18f7a16d76de02cae3de7fcb6d4e875d